Sources/Error.swift

```diff
@@ -1,8 +1,8 @@
-public protocol Error: ErrorProtocol {
+public protocol S4Error: Error {
```

Comment:
Lack of autocomplete? How so? Well, if not `Error` it should at least be `HTTPError` since it's what it is. We should go for meaningful names. I don't think adding a prefix for just the sake of disambiguation is the best option. What do you think? 😊